projects/experimental/components/avatar-labelled/avatar-labeled.styles.less

```diff
@@ -0,0 +1,19 @@
+.tui-avatar-labelled {
+    display: flex;
+    width: 3.5rem;
+    box-sizing: content-box;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+    line-height: 0.895rem;
+    font-size: 0.75rem;
+    padding: 0 0.5rem;
+
+    tui-avatar:first-child {
```

Comment:
Please remove the `:first-child`, it might not always be the case.

Reply:
fixed